website/content/docs/upgrading/identity-deduplication.mdx

```diff
@@ -0,0 +1,406 @@
+
+ 1. Ensure all clusters (primary and secondaries) are upgraded to 1.19.0 or
+    higher.
+ 1. On the primary, address each duplicate reported using the type-specific
+    guidance below.
+ 2. On any Performance Replication secondaries, address each _local alias_ duplicate reported using the guidance below.
+ 3. Activate the `force-identity-deduplication` activation flag.
+
+#### Without Vault replication
+
+ 1. Ensure the cluster is upgraded to 1.19.0 or higher.
+ 1. Address each duplicate reported using the type specific guidance below.
+ 2. Activate the `force-identity-deduplication` activation flag.
+
+### Resolving different-case entity alias duplicates
+
+```
+[WARN]  identity: 2 different-case entity alias duplicates found (potential security risk)
+[WARN]  identity: entity-alias "alias-case" with mount accessor "auth_userpass_34aca7ec" duplicates 1 others: id=df3568a4-3b65-4104-9481-1129ecbed72f canonical_id=5f013d99-a6c7-9a00-6ad5-4ad724b14f60 force_deduplication="would merge into entity 7da76b0d-fe9b-a125-3362-2a8ff055dcf8"
+[WARN]  identity: entity-alias "alias-cAsE" with mount accessor "auth_userpass_34aca7ec" duplicates 1 others: id=2992253b-1e99-4e47-b6f9-afb0c7cedf7a canonical_id=7da76b0d-fe9b-a125-3362-2a8ff055dcf8 force_deduplication="would merge others into this entity"
+[WARN]  identity: entity-alias "alias-case" with mount accessor "auth_userpass_a555989a" duplicates 1 others: id=0e4bd46e-a868-4dd4-a34a-cb73f097e3a5 canonical_id=1ce06951-e2fd-0923-8ae0-a0e2c6c2378b force_deduplication="would merge into entity 37f41fcf-5e15-f13f-248d-9fa8405b1ecd"
+[WARN]  identity: entity-alias "alias-cAsE" with mount accessor "auth_userpass_a555989a" duplicates 1 others: id=6cafe546-665c-4bd0-a4fd-5e699ba413c1 canonical_id=37f41fcf-5e15-f13f-248d-9fa8405b1ecd force_deduplication="would merge others into this entity"
+[WARN]  identity: end of different-case entity-alias duplicates
+```
+
+These duplicates are caused by auth mounts that have somehow (due to historical
+bugs) been allowed to create aliases that differ only in their case. One known
+way this can happen is if you use an external auth method such as LDAP and for
+some reason have an LDAP system that will sometimes return usernames normalized
+to uppercase and sometimes to lower or mixed case.
+
+By default Vault would match these in a case-insensitive way and so both would
+be considered the same alias. Past bugs have left some users with these
+different-case duplicates in storage though. To avoid the potential security
+impact of merging them, Vault since version 0.11.8 attempts to honor the past
+behavior by switching the identity engine to a case-sensitive matching mode to
+avoid accidentally merging two entities that might not be the same user.
+
+<Note> These are the most critical duplicates to resolve because there may be
+  security impact when enabling the `force-identity-deduplication` flag. </Note>
```

Comment:
As in, because of potential increases in permissions that were given to one entity's policy but not the other? Is it worth spelling out somewhere (maybe we do, still reading) that activating the flag should be paired with a review of the policies that apply to the detected duplicates shown in the server logs?

Reply:
Yeah, this whole guide is basically trying to say what you said? The overview of the process at the start was my attempt at introducing the idea that you need to review the report first before using the feature but if that wasn't clear enough then I'll take another look.